packages/camera/example/test_driver/camera.dart

```diff
@@ -0,0 +1,150 @@
+import 'dart:async';
+import 'dart:io';
+import 'dart:ui';
+
+import 'package:flutter/painting.dart';
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:camera/camera.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:video_player/video_player.dart';
+
+void main() {
+  final Completer<String> completer = Completer<String>();
+  Directory testDir;
+  enableFlutterDriverExtension(handler: (_) => completer.future);
+
+  setUpAll(() async {
+    final Directory extDir = await getApplicationDocumentsDirectory();
```

Comment:
Wouldn't it be better to use `getTemporaryDirectory` so that you wouldn't have to delete them all each time?

Reply:
`getTemporaryDirectory()` still needs to be deleted by us but that does make more sense. Changed.